ruff.toml

```
@@ -55,7 +56,7 @@ select = [
    "NPY",
    "AIR",
 ]
-ignore = ["E501", "E722", "F821"]
+ignore = ["E501", "E722", "F821", "SIM102", "SIM113", "SIM115", "SIM116", "SIM117"]
```

It might also help to set them like this to avoid conflicts in subsequent PRs,
```suggestion
ignore = [
    "E501",
    "E722",
    "F821",
    "SIM102",
    "SIM113",
    "SIM116",
]
```